components/movementsensor/rtkutils/gpsrtk.go

```diff
@@ -1,290 +1,120 @@
-// Package rtkutils defines a gps and an rtk correction source
-// which sends rtcm data to a child gps
-// This is an Experimental package
+// Package rtkutils implements functions that are used in the gpsrtkserial and gpsrtkpmtk.
```

Comment:
I think we should also rename this file to vrs.go ? Since it's only dealing with virtual reference stations and the connection logic to handle them, and no longer instatiating or creating a GPS object and fulfill the movement sensor interface methods. 

Looks really good!